### PR TITLE
[PFX-186] - jest refactor @gasket/plugin-nextjs

### PR DESCRIPTION
**Pull request by mmason2-godaddy**
## Summary
- jest refactor @gasket/plugin-nextjs

## Changelog
- jest refactor @gasket/plugin-nextjs

## Test Plan
Screen Shot 2023-01-19 at 1 23 18 PM

### CODE REVIEW

File: packages/gasket-plugin-nextjs/package.json [hunk 1]
```diff
@@ -109,5 +104,8 @@
     "presets": [
       "@babel/preset-react"
     ]
+  },
+  "jest": {
+    "rootDir": "test"
   }
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-nextjs/package.json:110]: Why is this needed?

File: packages/gasket-plugin-nextjs/test/config.test.js [hunk 2]
```diff
@@ -239,31 +226,31 @@ describe('createConfig', () => {
...
 
     describe('built-ins', () => {
 
       it('configures SASS loader', () => {
         result = config.webpack(webpackConfig, data);
 
-        assume(result.module.rules.some(rule => rule.test.test('bogus.scss')));
-        assume(result.module.rules.some(rule => rule.test.test('bogus.sass')));
+        expect(result.module.rules.some(rule => rule.test.test('bogus.scss'))).toBeFalsy();
+        // expect(result.module.rules.some(rule => rule.test.test('bogus.sass')));
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-nextjs/test/config.test.js:239]: Still working on this expect?

File: packages/gasket-plugin-nextjs/package.json (hunk 1 above)
Comment (mmason2-godaddy) [reply packages/gasket-plugin-nextjs/package.json:110]: Because it's going into the generator directory

File: packages/gasket-plugin-nextjs/test/config.test.js (hunk 2 above)
Comment (mmason2-godaddy) [reply packages/gasket-plugin-nextjs/test/config.test.js:239]: No forgot to delete it. This was doing nothing previously IMO.

Comment (mmason2-godaddy) [reply packages/gasket-plugin-nextjs/test/config.test.js:239]: It had no assertions

File: packages/gasket-plugin-nextjs/package.json (hunk 1 above)
Comment (agerard-godaddy) [reply packages/gasket-plugin-nextjs/package.json:110]: 👍 got it

**Review by agerard-godaddy — APPROVED** [review]
